oleesir/diaryReload
# ft(add-note-template) : implement add-note-template

#### What does this PR do?
This pull request adds an add note template

#### Description of Task to be completed?
add add note template

#### How should this be manually tested?
Navigate to the UI folder and open add-note.html in your browser
You will see an add note template

#### What are the relevant pivotal tracker stories?
N/A
#### Any background context you want to add?
#160876513
<img width="1280" alt="screen shot 2018-10-01 at 1 09 10 pm" src="https://user-images.githubusercontent.com/17679511/46289802-011e8d80-c582-11e8-8534-5ba985d34962.png">

#### Screenshots
